docker/images/java-agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM eclipse-temurin:17-alpine
+FROM eclipse-temurin:22-alpine
```

Comment:
This Java agent is supposed to run Java 17, does it not work on your architecture?

Reply:
I think I need to add another agent that runs Java 21